source/renderer/app/containers/wallet/dialogs/settings/WalletPublicKeyQRCodeDialogContainer.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+import React, { Component } from 'react';
+import { inject, observer } from 'mobx-react';
+import WalletPublicKeyQRCodeDialog from '../../../../components/wallet/settings/WalletPublicKeyQRCodeDialog';
+import type { InjectedProps } from '../../../../types/injectedPropsType';
+
+type Props = InjectedProps;
+
+@inject('actions', 'stores')
+@observer
+export default class WalletPublicKeyQRCodeDialogContainer extends Component<Props> {
+  static defaultProps = { actions: null, stores: null };
+
+  render() {
+    const { actions, stores } = this.props;
+    const { wallets } = stores;
+    const activeWallet = wallets.active;
+    const activeWalletPublicKey = wallets.activePublicKey;
```

Comment:
You can shorten these 2 lines to 1 line:

`const { active, activePublicKey } = wallets;`